packages/contracts/test/voting/addresslist-voting-setup.ts

```diff
@@ -94,9 +95,7 @@ describe('AddresslistVotingSetup', function () {
       const dataHRABI =
```

Comment:
why is this here ?

Reply:
It seems this checks don't make sense anymore.

Reply:
amended